app/master/cluster_master.py

```diff
@@ -196,7 +196,6 @@ def update_slave_last_heartbeat_time(self, slave) -> bool:
         """
         Updates last heartbeat time for the slave if it is alive.
         :type slave: Slave
-        :return: A boolean indicating whether last heartbeat time was called or not
         """
         if slave.is_alive():
             slave.update_last_heartbeat_time()
```

Comment:
If this code was actually returning `true` when the hearbeat tims is updated then it could be simplified to:

    return slave.is_alive() and slave.update_last_hearbeat_time()